design/ceph/object/zone.md

```diff
@@ -134,6 +145,8 @@ The following variables can be configured in the ceph-object-zone resource.
 
 - `customEndpoints`:  Specify the endpoint(s) that will accept multisite replication traffic for this zone. You may include the port in the definition if necessary. For example: "https://my-object-store.my-domain.net:443".
 
+- `preservePoolsOnDelete`: If it is set to 'true' the pools used to support the zone will remain when the ceph object zone CR will be deleted. This is a security measure to avoid accidental loss of data. It is set to 'true' by default. If not specified is also deemed as 'true'.
```

Comment:
```suggestion
- `preservePoolsOnDelete`: If it is set to 'true' the pools used to support the zone will remain when the CephObjectZone is deleted. This is a security measure to avoid accidental loss of data. It is set to 'true' by default. If not specified it is also deemed as 'true'.
```